content/posts/testing-farm-triggering.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Running tests with builds from another PR"
+date: 2022-12-21
+weight: 93
+---
+
+Do you contribute to projects which depend on each other?
+Would you like to test changes spanning multiple repositories together before merging them to the main branch?
+Then look no further, Packit's new feature of the Testing Farm integration is what you are looking for!
+
+### How it works
+
+To enable such testing, there is no additional configuration required in your `packit.yaml`, the typical [Testing Farm configuration](/docs/testing-farm/) is sufficient.
+Once you open a pull request with some changes, tests are going to run as usual with all dependencies being installed based on the test definition, e.g. from Fedora's repositories.
+To trigger tests with builds from a pull request in another repository, add a comment to the pull request of the form:
+
+    /packit test <namespace>/<repo>#pr-id>
```

Comment:
```suggestion
    /packit test <namespace>/<repo>#pr-id
```

It could be that the `>` was too much?

Reply:
Good catch. I think that `/packit test <namespace>/<repo>#<pr-id>` was what I was aiming for but missed `<`